types/jasmine-node/index.d.ts

```diff
@@ -2,42 +2,42 @@
 // Project: https://github.com/mhevery/jasmine-node
 // Definitions by: Sven Reglitzki <https://github.com/svi3c>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
-// TypeScript Version: 2.8
+// TypeScript Version: 3.9
```

Comment:
```suggestion
// TypeScript Version: 3.9
```
why this change? What's the new feature requiring this? The package itself is maintenance mode:
https://github.com/mhevery/jasmine-node#project-status

Reply:
No idea. It seemed to be required for tests to pass. Something about merging the two global jasmine namespaces?
Yes I agree the project is in maintenance mode which is why I'm ok with making this somewhat breaking change.

Reply:
```diff
diff --git a/types/jasmine-node/index.d.ts b/types/jasmine-node/index.d.ts
index 8000a8a986..71889e71ca 100644
--- a/types/jasmine-node/index.d.ts
+++ b/types/jasmine-node/index.d.ts
@@ -2,8 +2,6 @@
 // Project: https://github.com/mhevery/jasmine-node
 // Definitions by: Sven Reglitzki <https://github.com/svi3c>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
-// TypeScript Version: 3.9
-
 import "jasmine";
 
 declare global {
@@ -16,7 +14,7 @@ declare global {
 
         interface ExecuteSpecsOptions {
             specFolders: string[];
-            onComplete?: (runner: jasmine.Runner) => void;
+            onComplete?: (runner: Runner) => void;
             isVerbose?: boolean;
             showColors?: boolean;
             teamcity?: string | boolean;
```
this will fix tests, `jasmine` v1 is not a proper module)

Reply:
(removing version will make it compatible and tested with all TS tested via DT CI)